modules/virt-about-eus-updates.adoc

```diff
@@ -6,7 +6,14 @@
 [id="virt-about-eus-updates_{context}"]
 = About EUS-to-EUS updates
 
-Every even-numbered minor version of {product-title}, including 4.10 and 4.12, is an Extended Update Support (EUS) version. However, because Kubernetes design mandates serial minor version updates, you cannot directly update from one EUS version to the next. You must first update to the each z-stream release of the odd-numbered minor version before you can proceed to the target EUS version.
+Every even-numbered minor version of {product-title}, including 4.10 and 4.12, is an Extended Update Support (EUS) version. However, because Kubernetes design mandates serial minor version updates, you cannot directly update from one EUS version to the next. 
+
+After you update from the source EUS version to the next non-EUS minor version, you must sequentially update {VirtProductName} to all z-stream releases of that minor version that are on your update path. When you have upgraded to the latest applicable z-stream version, you can update {product-title} to the target EUS minor version. 
```

Comment:
Suggestions:
```suggestion
After you update from the source EUS version to the next non-EUS minor version, you must then sequentially update {VirtProductName} to all z-stream releases of that minor version that are on your update path. When you have upgraded to the latest applicable z-stream version, you can then update {product-title} to the target EUS minor version. 
```

Reply:
The "non-EUS minor version" wording is a little confusing to me but it may just be my unfamiliarity with the subject. Would a non-EUS minor version be synonymous with an odd-numbered version?